cost/azure/csp/cheaper_regions/README.md

```diff
@@ -0,0 +1,28 @@
+## Azure Cheaper Regions Policy
+
+### What it does
+
+This Policy Template determines regions have cheaper alternatives by specifying the expensive region name and the cheaper region name for analysis
```

Comment:
Add the word `which` in between determine and regions.  `This Policy Template determines which regions ....`